packages/vscode-extension/test/suite/envChecker/utils/common.ts

```diff
@@ -30,10 +31,51 @@ export function assertPathEqual(actual: string, expected: string) {
 }
 
 export async function getExecutionPolicyForCurrentUser(): Promise<string> {
-  const policy = await cpUtils.executeCommand(undefined, logger, undefined, "powershell.exe", "-Command", "Get-ExecutionPolicy", "-Scope", "CurrentUser");
+  const policy = await cpUtils.executeCommand(
+    undefined,
+    logger,
+    undefined,
+    "powershell.exe",
+     "-Command",
+    "Get-ExecutionPolicy",
+    "-Scope",
+    "CurrentUser"
+  );
   return policy.trim();
 }
 
 export async function setExecutionPolicyForCurrentUser(policy: string): Promise<void> {
-  await cpUtils.executeCommand(undefined, logger, undefined, "powershell.exe", "-Command", "Set-ExecutionPolicy", "-Scope", "CurrentUser", policy);
+  await cpUtils.executeCommand(
+    undefined,
+    logger,
+    undefined,
+    "powershell.exe",
+    "-Command", 
+    "Set-ExecutionPolicy",
+    "-Scope",
+    "CurrentUser",
+    policy
+  );
+}
+
+export async function createTmpDir(): Promise<[string, () => void]> {
```

Comment:
nit: the keyword async is not needed here.

Reply:
Fixed